test/to-govspeak.test.js

```diff
@@ -112,3 +112,18 @@ it('extracts headers from lists', () => {
   `
   expect(toGovspeak(html)).toEqual('## Item 1\n    \n### Item 2')
 })
+
+it('removes nested links when link markdown text is wrapped in an element', () => {
+  const html = `
+    [correct link markdown](https://www.gov.uk/)
```

Comment:
Since this has split into two tests I think this line has become somewhat confusing. 

I'm not sure if we need it in a test of it's own - it seems to be implicitly covered by https://github.com/alphagov/paste-html-to-govspeak/blob/87831980d564cfb9f3eab00302ca03e8a853bbea/test/to-govspeak.test.js#L10-L11 which is probably enough for now as we don't actually expect this to have broken.

Reply:
I've removed that test and mentioned in the commit that this situation happens when Google docs autolinks URLs.